pkg/broker/ingress/multi_topic_decouple_sink.go

```diff
@@ -69,6 +70,12 @@ type multiTopicDecoupleSink struct {
 
 // Send sends incoming event to its corresponding pubsub topic based on which broker it belongs to.
 func (m *multiTopicDecoupleSink) Send(ctx context.Context, broker types.NamespacedName, event cev2.Event) protocol.Result {
+	// Check to see if there are any triggers interested in this event. If not, no need to send this
```

Comment:
I have thought about the positioning of this block wrt to the block under it.

Right now if a broker is setup (even incorrectly, e.g., no topic) without targets, ingress will accept and acknowledge all incoming events.

If we move this block under the one below, it will at least check to see the topic exists before accepting events.

I'm open to either approach, and would appreciate it if anyone else has other thoughts.

Reply:
I think putting this below the `topic, err := m.getTopicForBroker(ctx, broker)` is better. `getTopicForBroker` should usually be a faster operation than applying the filters.

Reply:
Done.